### ISSUE TITLE
Availability trained models

### ISSUE THREAD

**Issue by lorenzoFabbri** [jgmeyerucsd/drug-class#11]
Would it be possible to share also the trained models (weights)? Thank you.

**Comment by jgmeyerucsd** [comment]
Sure but those files are over the github size limit. Which models are you interested in? I think I only saved a subset of them but can retrain tomorrow and save.

**Comment by lorenzoFabbri** [comment]
I guess the 5-class model is fine, if you have it.

**Comment by agitter** [comment]
@jgmeyerucsd if the saved models are very large, you could consider making another Zenodo repository for them.  That's usually faster than setting up git LFS (Large File Storage).

**Comment by jgmeyerucsd** [comment]
Will try to do this tomorrow

**Comment by jgmeyerucsd** [comment]
started training the model - will upload to zenodo tomorrow morning and send a link

**Comment by jgmeyerucsd** [comment]
Here is a link to a few of the trained model weights from the single-label 5 class CNN model: 
https://doi.org/10.5281/zenodo.3404012

One of them uses only a single example as the validation set (couldn't figure out how to use all data for training), and the other two use 20% of the data for validation set as described in the paper. The model that uses all the data for training should do better than the other two. See the zenodo description for more details

@lorenzoFabbri please let me know if that works for you or if you need any more help
